README.md

```diff
@@ -99,19 +99,6 @@ Run `poetry shell` to enter the virtual environment and then run `pytest`.
 To have the test suite watch for changes in code and run automatically, use
 pytest-watch by running `ptw`.
 
-## Code style
```

Comment:
Should there be a link to CONTRIBUTING.md in the README now that this section was removed?